src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" action="hey" method="post">
+      <label>
+        <input 
+        name="bigSearch"
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search big-search"
+        data-qa="keypress"
+      >
```

Comment:
fix formatting everywhere

```suggestion
        <input 
          name="bigSearch"
          type="search" 
          placeholder="Try &quot;Los Angeles&quot;"
          class="search big-search"
          data-qa="keypress"
      >
```